test/functional/tiertwo_governance_sync_basic.py

```diff
@@ -159,12 +175,17 @@ def check_proposal_existence(self, proposalName, proposalHash):
             assert_equal(proposals[0]["Hash"], proposalHash)
 
     def check_vote_existence(self, proposalName, mnCollateralHash, voteType):
-        for node in self.nodes:
+        for i in range(0, len(self.nodes)):
+            node = self.nodes[i]
             votesInfo = node.getbudgetvotes(proposalName)
             assert(len(votesInfo) > 0)
-            voteInfo = votesInfo[0]
-            assert_equal(voteInfo["mnId"], mnCollateralHash)
-            assert_equal(voteInfo["Vote"], voteType)
+            found = False
+            for voteInfo in votesInfo:
+                if (voteInfo["mnId"] == mnCollateralHash) :
+                    assert_equal(voteInfo["Vote"], voteType)
+                    found = True
+            assert_true(found, "Error checking vote existence in node " + str(i))
```

Comment:
nit: could rewrite all this function in a single pythonic one-liner as:
```python
assert_equal([x["Vote"] for node in self.nodes for x in node.getbudgetvotes(proposalName)
                      if x["mnId"] == mnCollateralHash], [voteType] * len(self.nodes))
```

Reply:
would be nice if we could print in this magic line which node wasn't able to sync as well.